src/main/java/com/odysseusinc/arachne/datanode/service/impl/AchillesServiceImpl.java

```diff
@@ -333,11 +336,13 @@ public void pullAchillesData(AchillesJob job) {
                 if (LOGGER.isDebugEnabled()) {
                     LOGGER.debug("Achilles data has collected at {}", tempDir);
                 }
-                retryTemplate.execute((RetryCallback<Void, Exception>) retryContext -> {
+                if (Objects.equals(FunctionalMode.NETWORK, dataNodeService.getDataNodeMode())) {
```

Comment:
What's the reason to call the method at all since its pulling data only to be sent to Central?